config/distscript_embedded.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+chmod u+w $2
```

Comment:
Also, do you need to `chmod u-w` on the `$distdir` at the end of this script?  I.e., do you need to un-do the `chmod` changes that you did?  I don't know if this has an effect on the generated tarball or not.